<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BasicBoss C64 PRG-Compiler</string>
  </property>
  <property name="windowIcon">
   <iconset resource="basicboss.qrc">
    <normaloff>:/img/img/c64_logo.png</normaloff>:/img/img/c64_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(190, 190, 190);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="basicboss.qrc">:/img/img/Commodore_64_logo.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>52</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Quelldatei:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leSource">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnFileOpen">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>21</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="basicboss.qrc">
          <normaloff>:/img/img/Open.png</normaloff>:/img/img/Open.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>52</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Zieldatei:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leTarget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="minimumSize">
         <size>
          <width>321</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 170, 0);</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="autoFormatting">
         <set>QTextEdit::AutoAll</set>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
        <property name="openLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnCompile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>271</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Compilieren</string>
        </property>
        <property name="icon">
         <iconset resource="basicboss.qrc">
          <normaloff>:/img/img/compiler.png</normaloff>:/img/img/compiler.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnStart">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="basicboss.qrc">
          <normaloff>:/img/img/VICE.png</normaloff>:/img/img/VICE.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProjekt">
    <property name="title">
     <string>Projekt</string>
    </property>
    <addaction name="actionLaden"/>
    <addaction name="separator"/>
    <addaction name="actionCompilieren"/>
    <addaction name="separator"/>
    <addaction name="actionStart_im_Emulator"/>
    <addaction name="separator"/>
    <addaction name="actionEinstellungen"/>
    <addaction name="separator"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionBasicBoss_Handbuch"/>
    <addaction name="separator"/>
    <addaction name="actionInfo_zum_Programm"/>
    <addaction name="actionInfo_zu_Qt"/>
   </widget>
   <addaction name="menuProjekt"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLaden"/>
   <addaction name="actionCompilieren"/>
   <addaction name="separator"/>
   <addaction name="actionStart_im_Emulator"/>
   <addaction name="separator"/>
   <addaction name="actionEinstellungen"/>
   <addaction name="separator"/>
   <addaction name="actionBasicBoss_Handbuch"/>
   <addaction name="actionInfo_zum_Programm"/>
   <addaction name="actionInfo_zu_Qt"/>
   <addaction name="separator"/>
   <addaction name="actionBeenden"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLaden">
   <property name="icon">
    <iconset resource="basicboss.qrc">
     <normaloff>:/img/img/Open.png</normaloff>:/img/img/Open.png</iconset>
   </property>
   <property name="text">
    <string>BASIC-Programm laden...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCompilieren">
   <property name="icon">
    <iconset resource="basicboss.qrc">
     <normaloff>:/img/img/compiler.png</normaloff>:/img/img/compiler.png</iconset>
   </property>
   <property name="text">
    <string>Compilieren</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionStart_im_Emulator">
   <property name="icon">
    <iconset resource="basicboss.qrc">
     <normaloff>:/img/img/VICE.png</normaloff>:/img/img/VICE.png</iconset>
   </property>
   <property name="text">
    <string>Start im Emulator</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="icon">
    <iconset resource="basicboss.qrc">
     <normaloff>:/img/img/exit.png</normaloff>:/img/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionBasicBoss_Handbuch">
   <property name="icon">
    <iconset resource="basicboss.qrc">
     <normaloff>:/img/img/handbuch.png</normaloff>:/img/img/handbuch.png</iconset>
   </property>
   <property name="text">
    <string>BasicBoss Handbuch...</string>
   </property>
  </action>
  <action name="actionInfo_zum_Programm">
   <property name="icon">
    <iconset resource="basicboss.qrc">
     <normaloff>:/img/img/info.png</normaloff>:/img/img/info.png</iconset>
   </property>
   <property name="text">
    <string>Info zum Programm</string>
   </property>
  </action>
  <action name="actionInfo_zu_Qt">
   <property name="icon">
    <iconset resource="basicboss.qrc">
     <normaloff>:/img/img/qt5-logo.png</normaloff>:/img/img/qt5-logo.png</iconset>
   </property>
   <property name="text">
    <string>Info zu Qt</string>
   </property>
  </action>
  <action name="actionEinstellungen">
   <property name="icon">
    <iconset resource="basicboss.qrc">
     <normaloff>:/img/img/prefs.png</normaloff>:/img/img/prefs.png</iconset>
   </property>
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="basicboss.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionBeenden</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
